### DISCUSSION TITLE
Possibility of having pg_cron like scheduler?

### DISCUSSION THREAD

**Discussion by ramazanpolat** [ClickHouse/ClickHouse#33361] [Q&A] [answered]
Is there any plan to have a `pg_cron` like scheduler?

There is a post in [2022 Roadmap](https://github.com/ClickHouse/ClickHouse/issues/32513#issuecomment-1003554951) that addresses Batch Jobs but I'm not sure if they are the same thing.

**Comment by filimonov** [comment, marked as answer]
https://github.com/ClickHouse/ClickHouse/issues/33919
https://github.com/ClickHouse/ClickHouse/issues/30480
 https://github.com/ClickHouse/ClickHouse/issues/3685
